lib/explorer/series.ex

```diff
@@ -6118,6 +6118,29 @@ defmodule Explorer.Series do
     apply_series(series, :json_decode, [dtype])
   end
 
+  @doc type: :string_wise
```

Comment:
- should we cast to the dtype which is passed ?
- can we infer dtype from `LazySeries`/  `Series` ?